internal/security/secretstore/init.go

```diff
@@ -435,6 +436,21 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		}
 	}
 
+	// Configure Kong Admin API
+	lc.Info("Starting the Kong Admin API config file creation")
+
+	// Get an instance of KongAdminAPI and map the paths from configuration.toml
+	ka := NewKongAdminAPI()
+	ka.paths.config = kongAdminConfig.ConfigFilePath
+	ka.paths.template = kongAdminConfig.ConfigTemplatePath
+	ka.paths.jwt = kongAdminConfig.ConfigJWTPath
+
+	// Setup Kong Admin API loopback configuration
+	err = ka.Setup()
+	if err != nil {
+		lc.Errorf("failed to configure the Kong Admin API: %s", err.Error())
+	}
+
```

Comment:
Still bothers me that we are doing Kong setup in Secret Store Setup. Refresh us why tis has to be here?

Reply:
i think it is something related to file permission in the run-time Kong container.  Kong running as kong:nogroup permission eludes  `proxy-setup` writing/reading file on kong's container.

Reply:
Agree as well.  In the service sequencing diagram Kong can be one of the last services to run.  If the user chooses not to run Kong at all, for whatever reason, we can't hold up secretstore-setup.  If it is due to a permissions issue, perhaps we can fork this off as a background process?

Reply:
Looks this this code doesn't actually talk to Kong but just sets up the configuration files.  We should be sure to clarify that we aren't blocking secretstore-setup based on Kong service being up.

Reply:
Updated the comments.